postman/MIDAZ.postman_collection.json

```diff
@@ -1711,6 +1834,65 @@
 						}
 					},
 					"response": []
+				},
+				{
+					"name": "Accounts by by asset id",
```

Comment:
The postman file isn't up to date. Get the last version then add your changes, pls.

Reply:
updated